### PR TITLE
Add utility function that returns a cgroup controller folder

### PR DESCRIPTION
**Pull request by IlayRosenberg**
Implemented and exported `get_cgroup_mount(controller)`

### CODE REVIEW

File: granulate_utils/linux/cgroups/cgroup.py [hunk 1]
```diff
@@ -70,4 +72,29 @@ def find_v2_hierarchy() -> Optional[str]:
...
+        return v1_paths[controller]
+    if v2_path:
+        # v2 (unified) - all controllers are in a single unified hierarchy
+        return v2_path
+    # No cgroup mount of the requested controller
+    return None
+
+
+def is_valid_controller(controller: str) -> bool:
+    return controller in SUBSYSTEMS
```
Comment (d3dave) [inline granulate_utils/linux/cgroups/cgroup.py:100]: "valid" is a bit misleading. Perhaps "known"? This also does not check if the controller is actually in use on the system, so it's unclear what constitutes an "invalid" controller. A list of enabled subsystems is in `/proc/cgroups` if we actually care about that.

Comment (IlayRosenberg) [reply granulate_utils/linux/cgroups/cgroup.py:100]: Sure I can go for `known`. This isn't supposed to check if the controller is in use (mounted), but rather if the given string is a recognized cgroup controller

**Comment by Jongy** [comment]
@d3dave - will you take the full CR here?

**Comment by d3dave** [comment]
> @d3dave - will you take the full CR here?

Yes,  it already looks good. Just the func name and it's approved

**Comment by Jongy** [comment]
> > @d3dave - will you take the full CR here?
> 
> Yes, it already looks good. Just the func name and it's approved

Thanks

**Review by d3dave — APPROVED** [review]

**Pull request merged**